class.jetpack.php

```diff
@@ -7105,6 +7105,10 @@ function jetpack_user_col_style() {
 	 */
 	public static function is_akismet_active() {
 		if ( method_exists( 'Akismet' , 'http_post' ) || function_exists( 'akismet_http_post' ) ) {
+			$akismet_key = Akismet::verify_key( Akismet::get_api_key() );
```

Comment:
We're checking for both the method and the function to support older versions of Akismet... I think AK prior to version 3.0. As written, this line will fatal on older versions since it passes the test on L7107 https://plugins.trac.wordpress.org/browser/akismet/tags/2.6.1/akismet.php#L74

I think we can probably just decide to drop support for older versions—we're talking about sites that are running 4+ year-old software. I suggest updating L7107 to remove the `|| function_exists( 'akismet_http_post')` check and noting in an inline doc that we only support Akismet 3.0+

Reply:
Addessed in 369d93ed7b4ca0f11dc794793b9189202bd49293